src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+
+    public Lottery() {
+        colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        Random random = new Random();
```

Comment:
make random class level field
```suggestion
        Random random = new Random();
```

Reply:
not fixed